EDDiscovery/UserControls/HelpersScanDisplayUC/ScanDisplayNodes.cs

```diff
@@ -133,9 +133,29 @@ enum DrawLevel { TopLevelStar, PlanetLevel, MoonLevel };
 
                     nodelabels[1] = nodelabels[1].AppendPrePad(appendlabeltext, Environment.NewLine);
 
- //  nodelabels[1] = nodelabels[1].AppendPrePad("" + sn.ScanData?.BodyID, Environment.NewLine);
- 
-                    bool valuable = sc.EstimatedValue >= ValueLimit;
+//  nodelabels[1] = nodelabels[1].AppendPrePad("" + sn.ScanData?.BodyID, Environment.NewLine);
+                    
+                    var ev = sc.GetEstimatedValues();
+                    var PotValue = sc.EstimatedValue;
+
+                    if (sc.WasMapped == true && sc.WasDiscovered == true)
+                    {
+                         PotValue = ev.EstimatedValueMappedEfficiently;
+                    }
+                    else if (sc.WasMapped == true && sc.WasDiscovered == false)
+                    {
+                         PotValue = ev.EstimatedValueMappedEfficiently;
+                    }
+                    else if (sc.WasMapped == false && sc.WasDiscovered == true)
+                    {
+                         PotValue = ev.EstimatedValueFirstMappedEfficiently; 
+                    }
+                    else if (sc.WasMapped == false && sc.WasDiscovered == false)
+                    {
+                         PotValue = ev.EstimatedValueFirstDiscoveredFirstMappedEfficiently;
```

Comment:
If body wasn't mapped and discovered change PotValue to the value you get when you fly there and map it.